src/tool_fsharp/tool_fsc.csproj

```diff
@@ -0,0 +1,69 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.tasks" />
+
+  <PropertyGroup>
+    <VersionPrefix>$(CliVersionPrefix)</VersionPrefix>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>$(CLI_SharedFrameworkVersion)</RuntimeFrameworkVersion>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PublishDir>$(FSharpDirectory)</PublishDir>
+    <VersionSuffix>$(CommitCount)</VersionSuffix>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NetCore.App" Version="$(CLI_SharedFrameworkVersion)" />
+    <PackageReference Include="Microsoft.FSharp.Core.netcore" Version="$(CLI_FSharp_Version)" />
+    <PackageReference Include="Microsoft.FSharp.Compiler.netcore" Version="$(CLI_FSharp_Version)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="RunFsc.sh;RunFsc.cmd">
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+  </ItemGroup>
+
+  <Target Name="MakeFscRunnableAndMoveToPublishDir"
+          AfterTargets="Publish"
+          BeforeTargets="RemoveFilesAfterPublish">
+
+    <ItemGroup>
+      <AssetsToRemoveFromDeps Include="tool_fsc.dll" 
+                              SectionName="runtime"/>
+    </ItemGroup>
+
+    
+    <RemoveAssetFromDepsPackages DepsFile="$(PublishDir)/$(TargetName).deps.json"
+                                 SectionName="%(AssetsToRemoveFromDeps.SectionName)"
+                                 AssetPath="%(AssetsToRemoveFromDeps.Identity)" />
+   
+    <Copy SourceFiles="$(PublishDir)/runtimes/any/native/default.win32manifest;
```

Comment:
NIT: keep the formatting here consistent. If you are putting the file in the next line below, put it here as well. Or, move the first file up to match here.